api/openapi/core-command.yaml

```diff
@@ -61,34 +61,23 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Command'
-    CorrelatedRequest:
-      description: "Provides a correlation ID that can be used when grouping multiple requests into a single batch. For example, multiple commands can be issued at once. Each IssueCommandRequest has its own request identifier while the batch has an overall identifier. This allows for visibility into whether or not a response was obtained for every request in the batch."
+    BaseRequest:
+      description: "Defines basic properties which all use-case specific request DTO instances should support."
       type: object
       properties:
-        correlationId:
-          description: "Identifies the batch to which this request belongs. If used via REST, this value should be the same as that of  the ''X-Correlation-ID'' HTTP header."
-          type: string
-          format: uuid
-          example: "82eb2e26-0f24-48aa-ae4c-de9dac3fb9bc"
         requestId:
-          description: "Uniquely identifies this request apart from its peers in the batch. For implementation, recommend this value be generated by the type's constructor."
+          description: "Uniquely identifies this request apart from its peers in a batch. For implementation, recommend this value be generated by the type's constructor."
           type: string
           format: uuid
           example: "e6e8a2f4-eb14-4649-9e2b-175247911369"
       required:
-        - correlationId
         - requestId
-    CorrelatedResponse:
-      description: "Provides a correlation ID that can be used when grouping multiple responses into a single batch. For example, multiple commands can be issued at once. The responses can be returned as a batch or individually. It doesn't matter as long as both of these properties are correctly set because either way the caller will have visibility into whether or not a response was obtained for every request in the batch."
+    BaseResponse:
+      description: "Defines basic properties which all use-case specific response DTO instances should support"
       type: object
       properties:
-        correlationId:
-          description: "Identifies the batch to which this response belongs. If used via REST, this value should be the same as that of  the ''X-Correlation-ID'' HTTP header."
-          type: string
-          format: uuid
-          example: "82eb2e26-0f24-48aa-ae4c-de9dac3fb9bc"
         requestId:
-          description: "Uniquely identifies the request within the larger batch resulting in this response."
+          description: "Uniquely identifies the request within the larger batch that resulted in this response."
```

Comment:
`within the larger batch ` should be removed.  Commented here; applies elsewhere

Reply:
Addressed. Per Cloud's comment, YAML docs are now in a v2 folder.